genie-web/src/main/java/com/netflix/genie/web/dtos/ArchivedJobMetadata.java

```diff
@@ -0,0 +1,42 @@
+/*
+ *
+ *  Copyright 2019 Netflix, Inc.
+ *
+ *     Licensed under the Apache License, Version 2.0 (the "License");
+ *     you may not use this file except in compliance with the License.
+ *     You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *     Unless required by applicable law or agreed to in writing, software
+ *     distributed under the License is distributed on an "AS IS" BASIS,
+ *     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *     See the License for the specific language governing permissions and
+ *     limitations under the License.
+ *
+ */
+package com.netflix.genie.web.dtos;
+
+import com.netflix.genie.common.internal.dto.DirectoryManifest;
+import lombok.AllArgsConstructor;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.ToString;
+
+import java.net.URI;
+
+/**
+ * A simple POJO for a compound value of related information to a job archived location and files.
+ *
+ * @author tgianos
+ * @since 4.0.0
+ */
+@Getter
+@AllArgsConstructor
+@EqualsAndHashCode(doNotUseGetters = true)
+@ToString(doNotUseGetters = true)
+public class ArchivedJobMetadata {
+    private final String jobId;
+    private final DirectoryManifest manifest;
+    private final URI jobDirectoryRoot;
```

Comment:
Maybe `archiveBaseURL` would be more clear?

Either way, this makes assumptions about how the manifest is stored.
So maybe just `url` or `location`?

Reply:
Fair I'll change it this is the result of a copy paste from a reuse location